vars/withGithubNotify.groovy

```diff
@@ -26,36 +26,14 @@
     // block
   }
 */
-def call(Map params = [:], Closure body) {
-  def context = params.context
-  def description = params.containsKey('description') ? params.description : context
-  def redirect = params.tab ?: 'pipeline'
-  def isBo = params.get('isBlueOcean', false)
-
-  if (!context) {
-    error 'withGithubNotify: Missing arguments'
-  }
-
-  if (!redirect.startsWith('http')) {
-    if (isBo) {
-      redirect = getBlueoceanTabURL(redirect)
-    } else {
-      redirect = getTraditionalPageURL(redirect)
+def call(Map args = [:], Closure body) {
```

Comment:
This is just a wrapper but nothing else.

Do we want to move the apm call in here instead?